modules/ossm-threescale-manifests.adoc

```diff
@@ -6,6 +6,12 @@
 [id="ossm-threescale-manifests_{context}"]
 = Generating manifests from a deployed adapter
 
+[NOTE]
+====
+* `NAME` is an identifier you use to identify with the service you are managing with 3scale.
+* The `CREDENTIALS_NAME` reference is an identifier that corresponds to the `match` section in the rule configuration. This is automatically set to the `NAME` identifier if you are using the CLI tool.
+* Its value does not need to be anything specific: the label value should just match the contents of the rule. See link:https://github.com/3scale/3scale-istio-adapter/blob/v2.0.2/README.md#routing-service-traffic-through-the-adapter[Routing service traffic through the adapter] for more information.
```

Comment:
I am not sure if we should use links to a static version of upstream README as it may change with the new patch releases.

Reply:
This is unlikely to ever change as the adapter will see no more releases other than maintenance in already shipping products. The correct fix would probably be to link to the `v2.X` or the `v2.0.X` branches (which would guarantee this), but then given a big change not just the link but the surrounding text would also be outdated and need changes. Not something I'd consider important, but feel free to link instead to: `https://github.com/3scale/3scale-istio-adapter/blob/v2.X/README.md`